examples/example_plugin/example_plugin/templates/example_plugin/example_with_custom_template.html

```diff
@@ -0,0 +1,4 @@
+<script>
+    console.log("i'm injecting js");
+</script>
+{% extends 'extras/job.html' %}
```

Comment:
Perhaps something like:

```suggestion
{% extends 'extras/job.html' %}

{% block content %}
    <div class="row">
        <div class="col-md-12">
        This is custom content injected by this specific plugin-provided Job
        </div>
    </div>
    {{ block.super }}
{% endblock content %}

{% block javascript %}
    {{ block.super }}
    <script>
        console.log("This job from the example_plugin is injecting custom JavaScript");
    </script>
{% endblock javascript %}
```

Reply:
^done